using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,193 @@
+
+Create the _application user_:
+
+. Click *Users* in the *Manage* sidebar to view the user information for the realm. Click *Add User*.
+. Enter a valid *Username* and any additional optional information for the _application user_ and click *Save*.
+. Edit the user configuration. Click the *Credentials* tab in the user space and enter a password for the user. After the password has been confirmed you can click the *Reset Password* button to set the user password. A pop-up window will prompt for additional confirmation.
+. Click *Role Mappings* to list the realm and client role configuration. In *Available Roles*, add the JEE role created earlier. 
+
+
+=== Deploy the SSO-enabled JBoss EAP Image
+
+. Return to the OpenShift web console and click *Add to project* to list all of the default image streams and templates.
+. Use the *Filter by keyword* search bar to limit the list to those that match _sso_. You may need to click *See all* to show the desired application template.
+. Click the *_eap64-sso-s2i_* image to list all of the deployment parameters. Edit the configuration of the following SSO parameters:
++
+* *SSO_URI*: The SSO web server authentication address: _https://secure-sso-<project-name>.<hostname>/auth_
+* *SSO_REALM*: The SSO realm created for this procedure.
+* *SSO_USERNAME*: The name of the _realm management user_.
+* *SSO_PASSWORD*:  The password of the user.
+* *SSO_PUBLIC_KEY*: The public key generated by the realm. It is located in the *Keys* tab of the *Realm Settings* in the SSO console.
+* *SSO_BEARER_ONLY*: If set to *true*, the OpenID Connect client will be registered as bearer-only.
+* *SSO_ENABLE_CORS*: If set to *true*, the Keycloak adapter enables Cross-Origin Resource Sharing (CORS).
+. Click *Create* to deploy the JBoss EAP image.
+
+It may take several minutes for the JBoss EAP image to deploy. When it does, it can be accessed at:
+
+* _http://<application-name>-<project-name>.<hostname>/<app-context>_: for the web server, and
+* _https://secure-<application-name>-<project-name>.<hostname>/<app-context>_: for the encrypted web server, where <app-context> is one of app-jee, app-profile-jee, app-profile-jee-saml, or service depending on the example application.
+
+==== Alternate Deployments
+You can also create the client registration in the *Clients* frame of the *Configure* sidebar. Once a client has been registered, click the *Installation* tab and download the configuration *_.xml_*:
+
+* For OpenID Connect application sources, save the *Keycloak OIDC JBoss Subsystem XML* to `<file_path>/configuration/secure-deployments`.
+* For SAML application sources, save the *Keyclock SAML Wildfly/JBoss Subsystem* to `<file_path>/configuration/secure-saml-deployments`.
```

Comment:
In the two points above, there's two file paths, right? Should be marked up with:

`*_`<file_path>`/configuration/secure-deployments_*`